code/addons/themes/docs/api.md

```diff
@@ -195,12 +195,13 @@ setup((app) => {
 
 export const decorators = [
   withVuetifyTheme({
+    // These keys are the labels that will be displayed in the toolbar theme switcher  
+    // The values must match the theme keys from your VuetifyOptions
     themes: {
       light: 'light',
       dark: 'dark',
-      customTheme: 'myCustomTheme',
+      'high contrast': 'highContrast', 
     },
-    defaultTheme: 'customTheme', // The key of your default theme
+    defaultTheme: 'light', // The key of your default theme
   }),
-];
-```
+];```
```

Comment:
@justingish one small item for you to adjust when you have a moment. Can you move the triple ticks below the semi-colon? Turning it into:
````md

```js
// .storybook/preview.js

import { setup } from '@storybook/vue3'
import { registerPlugins } from '../src/plugins'
import { withVuetifyTheme } from './withVuetifyTheme.decorator'

setup((app) => {
  registerPlugins(app)
})

/* snipped for brevity */

export const decorators = [
  withVuetifyTheme({
    // These keys are the labels that will be displayed in the toolbar theme switcher
    // The values must match the theme keys from your VuetifyOptions
    themes: {
      light: 'light',
      dark: 'dark',
      'high contrast': 'highContrast',
    },
    defaultTheme: 'light', // The key of your default theme
  }),
]
```
````